Cache/IO.lean

```diff
@@ -268,9 +268,9 @@ def mkDir (path : FilePath) : IO Unit := do
 Given a path to a Lean file, concatenates the paths to its build files.
 Each build file also has a `Bool` indicating whether that file is required for caching to proceed.
 -/
-def mkBuildPaths (path : FilePath) : IO $ List (FilePath × Bool) := do
+def mkBuildPaths (path : FilePath) : IO $ FilePath × List (FilePath × Bool) := do
   let packageDir ← getPackageDir path
-  return [
+  return (packageDir, [
```

Comment:
This seems a bit unnecessary, why not just pass the `packageDir` in instead if you need it afterward?

Reply:
I think this is now moot, the data is now carried around in a reader monad instead.